DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/staticIndexes/storeFieldInIndex/StoreFieldsInIndex.java

```diff
@@ -0,0 +1,99 @@
+//region Usings
+
+import net.ravendb.client.documents.indexes.AbstractIndexCreationTask;
+import net.ravendb.client.documents.indexes.FieldStorage;
+import net.ravendb.client.documents.session.IDocumentQuery;
+import net.ravendb.client.documents.session.IDocumentSession;
+//endregion
+import net.ravendb.demo.common.DocumentStoreHolder;
+import java.util.Date;
+import java.util.List;
+
+public class StoreFieldsInIndex {
+    //region Demo
+    //region Step_1
+    public class OrdersQuantity_ByCompany extends AbstractIndexCreationTask {
+        //endregion
+
+        //region Step_2
+        public class IndexEntry {
+            private String company;
+            private int totalItemsOrdered;
+
+            public String getCompany() {
+                return company;
+            }
+
+            public void setCompany(String company) {
+                this.company = company;
+            }
+
+            public int getTotalItemsOrdered() {
+                return totalItemsOrdered;
+            }
+
+            public void setTotalItemsOrdered(int totalItemsOrdered) {
+                this.totalItemsOrdered = totalItemsOrdered;
+            }
+        }
+        //endregion
+
+        //region Step_3
+        public static class OrderProjectedDetails {
+            private Date orderedAt;
+            private int totalItemsOrdered;
+
+            public Date getOrderedAt() {
+                return orderedAt;
+            }
+
+            public void setOrderedAt(Date orderedAt) {
+                this.orderedAt = orderedAt;
+            }
+
+            public int getTotalItemsOrdered() {
+                return totalItemsOrdered;
+            }
+
+            public void setTotalItemsOrdered(int totalItemsOrdered) {
+                this.totalItemsOrdered = totalItemsOrdered;
+            }
+        }
+        //endregion
+
+        public OrdersQuantity_ByCompany() {
+            //region Step_4
+            map = "docs.Orders.Select(order => new { \n" +
+                "   Company = order.Company, \n" +
+                "   TotalItemsOrdered = Enumerable.Sum(order.Lines, orderLine => ((int) orderLine.Quantity)) \n" +
```

Comment:
should it be capitalized or not ?
i.e. `TotalItemsOrdered` or `totalItemsOrdered` ?

Reply:
Done